src/cursor-doc/token-cursor.ts

```diff
@@ -173,13 +177,17 @@ export class LispTokenCursor extends TokenCursor {
      *
      * @returns true if the cursor was moved, false otherwise.
      */
-    forwardSexp(skipComments = true): boolean {
+    forwardSexp(skipComments = true, skipMetadata = false): boolean {
```

Comment:
We should consider adding this to `backwardSexpr` too. They are, so far, always completely reciprocal. Of course, it might be the case that it doesn't make sense for backward movement, I haven't thought it through.

Reply:
I thought about that. So far this is only used by the debugger to get to the coordinates location, which is just a forward moving process. The editor commands still do not ignore metadata, so I don't see a use for it yet. I figured I would leave it be for now since it would serve no purpose.